web-server/sgx-wallet-impl/src/ported/kv_store/mod.rs

```diff
@@ -0,0 +1,69 @@
+//! Simple key-value store abstraction
+//!
+//! XXX port note: Key type changed from str to `[u8]`.
```

Comment:
```suggestion
//! XXX port notes: 
//!   - Key type changed from str to `[u8]`.
//!   - Added `KvStore::mutate`
//!   - Added `KvStore::try_insert`
```

Reply:
`mutate` and `try_insert` are part of the ported code, not port additions, so we probably don't need this?

Reply:
Ah did not realize you were porting from a different branch than main.